src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/CaseNoteMovementService.kt

```diff
@@ -68,7 +68,7 @@ class CaseNoteMovementService(
       .type(type)
       .subType(subType)
       .caseNoteText(note)
-      .agencyLocation(booking.location)
+      .agencyLocation(booking.createLocation)
```

Comment:
When creating a case note for a released prisoner being migrated into Restricted patients the location would be OUT - we want the prison instead